workitem/workitem.go

```diff
@@ -18,6 +18,8 @@ type WorkItem struct {
 	Version int
 	// the field values
 	Fields Fields `sql:"type:jsonb"`
+	// the position of workitem
+	Executionorder float64
```

Comment:
@aslakkuntsen why not Executionorder? I didnt keep it "order" because in postgres "order" is a keyword. and i am trying to avoid that word.